README.md

```diff
@@ -1,20 +1,61 @@
-# _Asciidoctor Jet_
 
-A ready-to-use jet black themed template for building static websites using [Asciidoctor](https://asciidoctor.org) with [features and fonts added](https://harshkapadia2.github.io/asciidoctor-jet/#_features) to the default Asciidoctor template.
+<br>
 
-[**Live demo and more details.**](https://harshkapadia2.github.io/asciidoctor-jet)
+<p align="center"><img width ="12%" src="./src/static/img/icon-512.png"></p>
 
-[Getting started](https://harshkapadia2.github.io/asciidoctor-jet/#_getting_started) is very straightforward!
+<i><h3 align="center">Asciidoctor Jet</h3></i>
 
-## Changelogs
+<br>
+
+<p align="center">
+  A ready-to-use jet black themed template for building static websites using <a href="https://asciidoctor.org">Asciidoctor</a>. 
+  <br>
+  With <a href="https://harshkapadia2.github.io/asciidoctor-jet/#_features">features and fonts added</a> to the default Asciidoctor template.
+</p>
+
+<br>
+
+<p align="center">
+  <b><a href="https://harshkapadia2.github.io/asciidoctor-jet">Explore the docs »</a></b>
+</p>
+<p align="center">
+  <a href="https://harshkapadia2.github.io/asciidoctor-jet/#_live_demos">View Live Demos</a>
+    &nbsp; · &nbsp; 
+  <a href="https://harshkapadia2.github.io/asciidoctor-jet/#_getting_started">Getting Started</a>
+    &nbsp; · &nbsp; 
+  <a href="https://github.com/HarshKapadia2/asciidoctor-jet/generate">Use this template</a>
+  
+</p>
+
+<br>
+
+### File Structure
+
+```
```

Comment:
Add something like this

```
### File Structure

The main content of the website resides in the [`src` directory](src).

src
┣ content
.
.
.
```

Reply:
Makes sense, will do!

Reply:
[Done](https://github.com/HarshKapadia2/asciidoctor-jet/pull/3/commits/97206e62f9c8d2dfb1a89c76dbfc2574d81ea45b).